csharp/src/Ice/SignalTransceiverStream.cs

```diff
@@ -59,14 +81,25 @@ protected ValueTask<T> WaitSignalAsync(CancellationToken cancel = default)
         T IValueTaskSource<T>.GetResult(short token)
```

Comment:
Why this doesn't have an access modifier?

Reply:
These don't need to be public.

Reply:
I mean we add `private` explicitly everywhere else